### PR TITLE
fix (api): do not call workflow.execute on add spawn info

### PR DESCRIPTION
**Pull request by yesnault**
Signed-off-by: Yvonnick Esnault <yvonnick.esnault@corp.ovh.com>

### CODE REVIEW

**Comment by fsamin** [comment]
humm, i think @sguiheux made some rework about this

**Comment by yesnault** [comment]
@fsamin sure, it's fixed

**Comment by fsamin** [comment]
As I said, i’m pretty sure that Steven already  refactor all of this and much more on another branch

**Comment by yesnault** [comment]
UT / IT / ITP OK

File: engine/api/workflow/dao_node_run_job.go [hunk 1]
```diff
@@ -141,8 +141,8 @@ func DeleteNodeJobRuns(db gorp.SqlExecutor, nodeID int64) error {
 	return err
 }
 
-//UpdateNodeJobRun updates a workflow_node_run_job
-func UpdateNodeJobRun(db gorp.SqlExecutor, store cache.Store, p *sdk.Project, j *sdk.WorkflowNodeJobRun) error {
+//UpdateNodeJobRunAndExecute updates a workflow_node_run_job then run workflow.execute
```
Comment (sguiheux) [inline engine/api/workflow/dao_node_run_job.go]: It's not the good way to fix it.
I already have a fix for that on a dev branch. I will make the PR today (i hope)

**Pull request closed without merging**